www/about/pricing.spt

```diff
@@ -83,7 +83,7 @@ subhead = _("Pricing")
 
     <p>Your payments cover hosting and other costs of doing business, and the
     rest goes to the <a
-    href="https://gratipay.com/Gratipay/members/">people</a> who are building
+    href="https://gratipay.com/explore/teams/">teams of people</a> who are building
```

Comment:
This change in wording alters the meaning of the sentence too drastically. The `/Gratipay/members` link is for the people actually on the Gratipay payroll (which is now gone, yes; #3433). `/explore/teams` is a listing of *all* teams on Gratipay, of which the Gratipay team itself is but one. "People" in this sentence refers to the people on the Gratipay team.